README.md

```diff
@@ -60,7 +60,7 @@ ethminer --help
 ```
 ### Examples connecting some pools
 
-Check our [samples](POOLEXSAMPLES_ETH.md) to see how to connect to different pools.
+Check our [samples](POOL_EXAMPLES_ETH.md) to see how to connect to different pools.
```

Comment:
This would create a broken link!!

Reply:
https://github.com/ethereum-mining/ethminer/pull/1202